cocos/renderer/CCGLProgramState.h

```diff
@@ -241,6 +241,9 @@ class CC_DLL GLProgramState : public Ref
 {
     friend class GLProgramStateCache;
 public:
+    /** x-studio365 spec, update node program state */
+    static GLProgramState* getPositionTextureColorGLProgramState(Texture2D* texture, bool noMVP = true);
+    static GLProgramState* getPositionTextureGrayGLProgramState(Texture2D* texture);
```

Comment:
I think it is not a good idea to add functions special for `position_texture_color` shaders.